### ISSUE TITLE
bug: Errors of Go scripts have the wrong line numbers

### ISSUE THREAD

**Issue by jaller94** [windmill-labs/windmill#644]
### Describe the bug

Because of the transformation of Go scripts, errors show incorrect line numbers.

This makes debugging Go scripts difficult.

### To reproduce

1. Create a Go script
2. Paste the code below.
3. Test it.
4. Check the line number mentioned in the error.

```go
import (
	"fmt"
)

func main(x string) (interface{}, error) {
	foo("")
  return x, nil
}
```

### Expected behavior

The error should be `mymod/main.go:6:2: undefined: foo` instead of `mymod/main.go:14:2: undefined: foo`.

### Screenshots

```
--- GO CODE EXECUTION ---

# command-line-arguments
mymod/main.go:14:2: undefined: foo

--- ERROR ---

Error during execution of the script
last 5 logs lines:

--- GO CODE EXECUTION ---

# command-line-arguments
mymod/main.go:14:2: undefined: foo
```

```
The result keys are: error 

{
    "error": "Error during execution of the script
last 5 logs lines:

--- GO CODE EXECUTION ---

# command-line-arguments
mymod/main.go:14:2: undefined: foo"
}
```

### Application version

windmill version (backend) v1.35.0-126-g5406a70

### Additional Context

Can the wrapper script import the user's script. Maybe the only transformation needed is to export the `main()` function.

**Comment by rubenfiszel** [comment]
As you rightly point out, I think we can probably fix #640 and this by importing the script rather than pasting it.

**Comment by rubenfiszel** [comment]
Solved by not wrapping the code anymore but using a separate file: [eabd835](https://github.com/windmill-labs/windmill/commit/eabd83580758121149b629285d8f4cb228c9a7ea)
